packages/lego-bricks/package.json

```diff
@@ -0,0 +1,69 @@
+{
+  "name": "@webkom/lego-bricks",
+  "version": "0.0.0",
+  "description": "Component library for lego and other Abakus projects",
+  "author": "webkom",
+  "license": "MIT",
+  "repository": {
+    "type": "git",
+    "url": "github.com/webkom/lego-webapp",
+    "directory": "packages/lego-bricks"
+  },
+  "type": "module",
+  "module": "dist/lego-bricks.js",
+  "main": "dist/lego-bricks.umd.cjs",
+  "style": "dist/style.css",
+  "files": [
+    "dist"
+  ],
+  "exports": {
+    ".": {
+      "import": "./dist/lego-bricks.js",
+      "require": "./dist/lego-bricks.umd.cjs"
+    },
+    "./dist/style.css": {
+      "import": "./dist/style.css",
+      "require": "./dist/style.css"
+    }
+  },
+  "scripts": {
+    "storybook": "storybook dev -p 6006",
+    "build": "vite build && yarn build:types",
+    "build:types": "tsc -d --emitDeclarationOnly",
+    "build:storybook": "storybook build",
+    "lint": "yarn eslint && yarn prettier && yarn types",
+    "eslint": "eslint 'src/**/*.{js,ts,tsx}'",
+    "prettier": "prettier 'src/**' './*.{json,js,ts,md}' --write",
+    "prettier:check": "prettier 'src/**' './*.{json,js,ts,md}' --list-different",
+    "types": "tsc --noEmit",
+    "test": "vitest",
+    "rest:run": "vitest run"
+  },
+  "dependencies": {
+    "react": "^18.2.0",
+    "react-dom": "^18.2.0",
```

Comment:
Wouldn't these be peer dependencies?

Reply:
I've never really understood peer dependencies, but I think you're right that peerDependencies is the "correct" way to do it for a component library.